### PR TITLE
update tabs on tiles

### PR DESCRIPTION
**Pull request by stackoverfloweth**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

### CODE REVIEW

File: src/pages/Flow/UpcomingRuns-Tile.vue
Comment (stackoverfloweth) [inline src/pages/Flow/UpcomingRuns-Tile.vue:113]: quick note here, I was frustrated that if there are runs in the "late" tab, even when I explicitly go back to the "upcoming" tab the system would continue to kick me back to "late" on a regular interval. I change this behavior to only programmatically switch the user to "late" tab if the user hasn't explicitly changed tabs

File: src/pages/Flow/UpcomingRuns-Tile.vue
Comment (stackoverfloweth) [inline src/pages/Flow/UpcomingRuns-Tile.vue:500]: this is the height of v-tabs

File: src/pages/Flow/Details-Tile.vue
Comment (stackoverfloweth) [inline src/pages/Flow/Details-Tile.vue:569]: this is the height of v-tabs

File: src/pages/Agents/SubmittableRuns.vue
Comment (stackoverfloweth) [inline src/pages/Agents/SubmittableRuns.vue:165]: quick note here, I was frustrated that if there are runs in the "late" tab, even when I explicitly go back to the "submittable" tab the system would continue to kick me back to "late" on a regular interval. I change this behavior to only programmatically switch the tab if the user hasn't explicitly changed tabs (same as UpcomingRunsTile)

File: src/pages/Flow/UpcomingRuns-Tile.vue
Comment (pleek91) [inline src/pages/Flow/UpcomingRuns-Tile.vue:195]: nitpick: make `new Date().getTime()` a const and use the same date for every flow run.

File: src/pages/FlowRun/Details-Tile.vue
Comment (pleek91) [inline src/pages/FlowRun/Details-Tile.vue:301]: I think since your `v-tabs` has a `v-if` on it you need to conditionally subtract `--v-tabs-height`

**Review by pleek91 — APPROVED** [review]
Handful of tiny suggestions. Looks great overall

**Review by zhen0 — CHANGES_REQUESTED** [review]
This looks nice @stackoverfloweth.  A few small comments from me (mainly about var vs const).  Also - could we include the upcoming/late runs tile on the dashboard in these changes?  It feels better to have that switch across the board.

File: src/pages/FlowRun/Details-Tile.vue
Comment (stackoverfloweth) [reply src/pages/FlowRun/Details-Tile.vue:301]: good catch, although upon closer inspection it seems that class isn't actually used in this template

File: src/pages/Flow/UpcomingRuns-Tile.vue
Comment (stackoverfloweth) [reply src/pages/Flow/UpcomingRuns-Tile.vue:195]: 🧠

**Review by zhen0 — COMMENTED** [review]
One tiny nitpick - there's a border above 'options' on the late run tab that I don't think we want/need. it's especially obvious in dark mode. 

This PR:

Screen Shot 2021-09-30 at 5 11 23 PM

Screen Shot 2021-09-30 at 5 11 36 PM

Current UI:

Screen Shot 2021-09-30 at 5 12 19 PM
Screen Shot 2021-09-30 at 5 12 32 PM

**Review by zhen0 — APPROVED** [review]

**Comment by zhen0** [comment]
Looks good to me - can we get another approval before merging to staging?  @pleek91 @whimsicallyson

**Comment by whimsicallyson** [comment]
my only question/comment is how it works if there are more tab options than the width of the card. does it overflow properly in the horizontal, or do we wend up with the same issue that we had with the old implementation?

**Comment by stackoverfloweth** [comment]
> my only question/comment is how it works if there are more tab options than the width of the card. does it overflow properly in the horizontal, or do we wend up with the same issue that we had with the old implementation?

@whimsicallyson Great point, yes it does overflow properly! That's built into v-tabs https://vuetifyjs.com/en/components/tabs/#dynamic-tabs.
You can see an example of this on the tile reference in the original bug (can have up to 4 tabs)

**Review by whimsicallyson — APPROVED** [review]

**Pull request merged**
